settings.gradle

```diff
@@ -15,9 +15,13 @@ rootProject.name = 'ballerina-persist'
 
 include ':checkstyle'
 include ':persist-ballerina'
+include ':persist-examples'
+
```

Comment:
Shall we remove the extra new line?
Please check all the places

Reply:
Fixed in [f7c186b](https://github.com/ballerina-platform/module-ballerina-persist/pull/5/commits/f7c186b06ba8d5a2f9183a613d63d2cec1f1bdf5)